docs/user-guide/commands/argocd_cluster_list.md

```diff
@@ -8,6 +8,24 @@ List configured clusters
 argocd cluster list [flags]
 ```
 
+### Examples
+
+```
+
+# List Clusters in Default "Wide" Format
+argocd cluster list
+
+# List Cluster via specifing the server
+argocd cluster list --server <ARGOCD_SERVER_ADDRESS>
+
+# List Clusters in JSON Format
+argocd cluster list -o json --server <ARGOCD_SERVER_ADDRESS>
+
+# List Clusters in YAML Format
+argocd cluster list -o yaml --server <ARGOCD_SERVER_ADDRESS>
```

Comment:
Can you also add an example for `-o server`?

Reply:
Yes 
I have added the example